bmeg_app/views/literature_support_view.py

```diff
@@ -48,90 +44,134 @@
-    # Filter for gene
-    df = base_df[base_df['geneID']==selected_gene].reset_index(drop=True)
-    # Generate table
-    df = lsu.occurances_table(df,'drug',['Compound','Lit Occurances'])
-    dashtable = dash_table.DataTable(
+    pub_table = dash_table.DataTable(
         id='export_pub_table',
-        data = df.to_dict('records'),
-        columns=[{"name": i, "id": i} for i in df.columns],
-        style_header={'text-align':'center','backgroundColor': 'rgb(230, 230, 230)','fontSize':styles['t']['size_font'],'fontWeight': 'bold','fontFamily':styles['t']['type_font']},
+        data = citation_counts[:10],
+        columns=[{"name": "Compound", "id": "compound"}, {"name" : "Citations", "id" : "citations"}],
+        style_header={'text-align':'left','backgroundColor': 'rgb(230, 230, 230)','fontSize':styles['t']['size_font'],'fontWeight': 'bold','fontFamily':styles['t']['type_font']},
         style_cell={'maxWidth':'100px','padding-left': '20px','padding-right': '20px'},
-        style_data={'whiteSpace':'normal','height':'auto','text-align':'center','fontFamily':styles['t']['type_font'],'fontSize':styles['t']['size_font']},
+        style_data={'whiteSpace':'normal','height':'auto','text-align':'left','fontFamily':styles['t']['type_font'],'fontSize':styles['t']['size_font']-2},
         style_data_conditional=[{'if': {'row_index': 'odd'},'backgroundColor': 'rgb(248, 248, 248)'}],
         style_table={'overflow':'hidden'},
-        style_as_list_view=True,
-        tooltip_data=[{column: {'value': str(value), 'type': 'markdown'} for column, value in row.items()} for row in df.to_dict('records')],
+        style_as_list_view=True,        #tooltip_data=[{column: {'value': str(value), 'type': 'markdown'} for column, value in row.items()} for row in df.to_dict('records')],
         tooltip_duration=None,
         page_size=10,
     )
-    return dashtable,
+
+    height = 250
+
+    fig= px.histogram(response, orientation="h", height=height, title="Response")
+    fig.update_layout(margin={'t':25, 'b':0, 'r': 0},
+            yaxis=dict(tickmode='linear'),
+            plot_bgcolor='white',
+            paper_bgcolor='white')
+    fig.update_layout(showlegend=False)
+    histogram = dcc.Graph(figure=fig)
+
+    '''pie chart compound taxonomy'''
+    associated_compounds = G.query().V(gene).out("g2p_associations").as_("a").out("compounds").as_("c").render(["$a._gid", "$c._gid"]).execute()
+    c = G.query().V(list(pd.DataFrame(associated_compounds)[1].unique())).render(["_gid", "taxonomy.superclass"]).execute()
+    compound_info = pd.DataFrame(c, columns=["id", "superclass"]).set_index("id")
+    fig = px.pie(compound_info, names="superclass", height=height, title="Compound Superclass")
+    fig.update_layout(showlegend=False, margin={'t':25, 'b':0, 'l':0, 'r': 0})
+    fig.update_traces(textposition='inside', textinfo='percent+label')
+    pie_chart = dcc.Graph(figure=fig)
+
+    return evidence_table, histogram, pie_chart, pub_table
```

Comment:
we should resize bar chart and pie chart to match the height of the table
![image](https://user-images.githubusercontent.com/40308471/95118027-67ed5900-06fe-11eb-9781-4b2f8651667b.png)